android/api-tests/build.gradle.kts

```diff
@@ -46,5 +46,8 @@ android {
 dependencies {
     implementation(project(":hybridcommon"))
     implementation(project(":hybridcommon-ui"))
-    testImplementation(libs.junit)
+}
+
+tasks.withType<AbstractTestTask>().configureEach {
```

Comment:
Did this to fix tests after the gradle major. I think it's ok but lmk if you have any concerns @RevenueCat/coresdk